apps/talk.sh

```diff
@@ -105,9 +105,15 @@ echo "TURN_CONF was successfully created"
 # Restart the TURN server
 check_command systemctl restart coturn
 
+# Warn user to open port
+msg_box "Open $TURN_PORT on your firewall or your TURN/STUN server won't work!
+After you hit OK the script will check for the firewall and eventually exit on failure.
+To run again the setup, after fixing your firewall:
+sudo wget $APP/talk.sh
+sudo bash talk.sh"
 
 # Check if the port is open
-check_open_port "$TURN_PORT" "$TURN_DOMAIN"
+check_open_port "$TU RN_PORT" "$TURN_DOMAIN"
 
```

Comment:
typo?

Reply:
I actually don't get this..? In the LE scritps we just do like [this](https://github.com/nextcloud/vm/blob/master/lets-encrypt/activate-ssl.sh#L47-L52) and [this](https://github.com/nextcloud/vm/blob/master/apps/collabora.sh#L36) in the Collabora script.

Reply:
Better to check if the port is open and exit if it's not IMHO. That's why I wanted to put the check in the beginning as we are using the Nextcloud URL anyway it doesn't matter if it's in the top of the script. I mean, why start installing stuff if it will fail in the end anyway? :)